vpr/src/base/read_options.cpp

```diff
@@ -1147,6 +1150,13 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         "If the implementation is illegal analysis can be forced by explicitly\n"
         "specifying the --analysis option.");
 
+    auto& FPGAInterchage_grp = parser.add_argument_group("FPGA Interchange options");
```

Comment:
Should we consider an option like ``--arch_format`` like the option ``--circuit_format`` which 
 is more flexible? 

https://docs.verilogtorouting.org/en/latest/vpr/command_line_usage/#cmdoption-vpr-circuit_format

Reply:
Yes, makes sense, I have added the corresponding change